test/extension/loadable_extension_optimizer_demo.cpp

```diff
@@ -21,6 +22,46 @@ using namespace duckdb;
 #include <sys/types.h>
 #include <arpa/inet.h>
 
+namespace duckdb {
+
+//! LogicalChunkGet represents a scan operation from a ChunkCollection
+class LogicalChunkGet : public LogicalOperator {
```

Comment:
Could we perhaps rewrite the test to use a `ColumnDataCollection` instead?

Reply:
Thanks! Works like a charm. I've updated it, let me know what you think